### PR TITLE
Add user profile screen

### LINKED ISSUE #980: Student Navigation to Self Profile
For students, the navigation menu should include a link to their profile — their user summary page -- as well as the link to dashboard

### PR DESCRIPTION
**Pull request by jrjohnson**
Users can no view their own profile with a link in the user menu.  I also split up the user and translations menu and made the whole thing responsive.

Fixes #980

### CODE REVIEW

**Comment by jrjohnson** [comment]
@saschaben I took some liberties restructuring the header adding a separate menu for translations and user actions.  Take a look and see what you think.  They change slightly in response to the size of the window as well.

Also see if we are including the right information on the my-profile page that is linked to from the user navigation in the header.

**Comment by saschaben** [comment]
👍  I think the header is great. the data appears to be correct, and for a first shot includes a decent baseline. We can always add further info if it seems appropriate.

the "student" label (class="is-student") should be a little more prominent though. right now it fades into the background above the roles box. Please bump it up in font size a couple points and bold it.

**Comment by jrjohnson** [comment]
@saschaben ready for another review.

**Comment by saschaben** [comment]
👍  💯

File: app/components/ilios-header.js [hunk 1]
```diff
@@ -1,17 +1,14 @@
 import Ember from 'ember';
 
 const { Component, computed, inject } = Ember;
-const { oneWay, equal } = computed;
 const { service } = inject;
 
 export default Component.extend({
   session: service(),
   currentUser: Ember.inject.service(),
```
Comment (stopfstedt) [inline app/components/ilios-header.js]: replace with `currentUser: service(),`.

File: app/components/ilios-header.js [hunk 2]
```diff
@@ -1,17 +1,14 @@
...
   i18n: Ember.inject.service(),
```
Comment (stopfstedt) [inline app/components/ilios-header.js]: replace with `i18n: service(),`.

**Comment by stopfstedt** [comment]
LGTM

**Pull request merged**